docker/psql/Jenkinsfile

```diff
@@ -0,0 +1,58 @@
+def dockerRegistry = "ghcr.io"
+def dockerRepo = "icgc-argo/backup-psql"
+def imageVersion = "0.1.0"
```

Comment:
It is generally not a good idea to have the version set explicitly in the Jenkinsfile, rather you want to derive it from the solution you are building/shipping. 

What we want is a label in the dockerfile itself 

```dockerfile 
LABEL org.icgc-argo.psql-backup.version = 0.1.0
```

And then that can be read in by the jenkinsfile.

Reply:
100% agree, good call. Version is now a label in Dockerfile. And Jenkinsfile pipeline step will retrieve version from label using `docker inspect`.